### PR TITLE
contribution details moved to contribution.md

### PR DESCRIPTION
**Pull request by lotharschulz**
adapted contribution hints and created contribution.md

### CODE REVIEW

**Comment by lotharschulz** [comment]
@CyberDem0n: pls check

File: README.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ For additional background info, see:
 Development Status
 ================
 
-Patroni is in active development and accepts contributions. See our `Contributing <https://github.com/zalando/patroni/blob/master/README.rst#contributing>`__ section below for more details. 
+Patroni is in active development and accepts contributions. See our `Contributing <https://github.com/zalando/patroni/blob/master/README.rst#contributing>`__ section below for more details.
```
Comment (CyberDem0n) [inline README.rst]: Shouldn't it be ' CONTRIBUTING.md' instead of 'README.rst#contributing'?

**Comment by lotharschulz** [comment]
true

**Comment by CyberDem0n** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

**Comment by feikesteenbergen** [comment]
👍

**Pull request merged**